infra/main.bicep

```diff
@@ -87,6 +91,14 @@ var iothub001Name = '${name}-iothub001'
 var eventhubNamespace001Name = '${name}-eventhub001'
 var streamanalytics001Name = '${name}-streamanalytics001'
 var streamanalyticscluster001Name = '${name}-streamanalyticscluster001'
+var logAnalytics001Name = '${name}-loganalytics001'
+var dataEmailActionGroup = '${name}-emailactiongroup'
+var synapsePipelineFailedAlertName = '${synapse001Name}-failedalert'
+var iothubFailedAlertName = '${iothub001Name}-failedalert'
+var eventhubnamespaceErrorAlertName = '${eventhubNamespace001Name}-erroralert'
+var streamanalyticsErrorAlertName = '${streamanalytics001Name}-erroralert'
+var dashboardName = '${name}-dashboard'
+var database001Name = 'Database001'
```

Comment:
Can you please check and be consistent with the naming convention here:
https://github.com/Azure/data-product-analytics/pull/123

Reply:
Just want to make sure that it is consistent. :)

Reply:
Made sure it is consistent.